test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -787,19 +787,19 @@ ZoneId=$FileType
                     $errorType = $null
                     if($null -ne $exception)
                     {
-                        $errorType = $exception.exception.getType()
+                        $errorType    = $exception.exception.getType()
```

Comment:
The change is unrelated with the PR - please revert.
Please don't make formatting changes in future.